x/concentrated-liquidity/keeper_test.go

```diff
@@ -430,18 +430,18 @@ func (s *KeeperTestSuite) AddBlockTime(timeToAdd time.Duration) {
 	s.Ctx = s.Ctx.WithBlockTime(s.Ctx.BlockTime().Add(timeToAdd))
 }
 
-func (s *KeeperTestSuite) TestValidatePermissionlessPoolCreationEnabled() {
+func (s *KeeperTestSuite) TestIsPermissionlessPoolCreationEnabled() {
 	// Normally, by default, permissionless pool creation is disabled.
 	// SetupTest, however, calls SetupConcentratedLiquidityDenomsAndPoolCreation which enables permissionless pool creation.
-	s.Require().NoError(s.App.ConcentratedLiquidityKeeper.ValidatePermissionlessPoolCreationEnabled(s.Ctx))
+	s.Require().True(s.App.ConcentratedLiquidityKeeper.IsPermissionlessPoolCreationEnabled(s.Ctx))
```

Comment:
It seems that the comment above is outdated.

Permissionless pool creation is now enabled in tests by default:
https://github.com/osmosis-labs/osmosis/blob/9382028fd94144ff5e2317cd8de52e80a1f575be/app/apptesting/concentrated_liquidity.go#L143

Would be great to update it